sources/dataset.py

```diff
@@ -306,6 +306,10 @@ def populate(self, output_dir, input_dir, nb_images=None, aggregate=False, label
 
 class ShapeDataset(Dataset):
 
+    SQUARE_COLOR = [0, 10, 10]
```

Comment:
I think a tuple structure is better because it's immutable.

Reply:
Done.